src/sub.c

```diff
@@ -178,37 +188,36 @@ _setDrainCompleteState(natsSubscription *sub)
                 sub->drainStatus = NATS_INVALID_SUBSCRIPTION;
         }
         sub->drainState |= SUB_DRAIN_COMPLETE;
-        natsCondition_Broadcast(sub->cond);
+
+        natsCondition_Broadcast(sub->drainCond);
     }
 }
 
-void
-natsSub_setDrainCompleteState(natsSubscription *sub)
+void natsSub_setDrainCompleteState(natsSubscription *sub)
 {
     natsSub_Lock(sub);
     _setDrainCompleteState(sub);
     natsSub_Unlock(sub);
 }
 
 // _deliverMsgs is used to deliver messages to asynchronous subscribers.
-void
-natsSub_deliverMsgs(void *arg)
+void natsSub_deliverMsgs(void *arg)
 {
-    natsSubscription    *sub        = (natsSubscription*) arg;
```

Comment:
Call me crazy, but I preferred the way I was doing before (fields aligned) :-) (but that's just personal preference).
Actually, I used even to align the `= ...` part of it, which had to be changed sometimes when adding a new field, etc..

Reply:
No problem.I checked my VSCode and don't see anything special that I have set that would override this behavior, so I would assume that you have something that makes that happen? I have something for the Editor that trims trailing spaces when saving a file.